marshal/consumer.go

```diff
@@ -19,6 +19,28 @@ import (
 	"github.com/dropbox/kafka/proto"
 )
 
+// CommitToken is a minimal structure that contains only the information necessary to
+// mark a message committed. This is done so that you can throw away the message instead
+// of holding on to it in memory.
+type CommitToken struct {
+	topic  string
+	partID int
+	offset int64
+}
+
+// Message is a container for Kafka messages.
+type Message proto.Message
```

Comment:
In your comment on the PR you said something about copying the message, but AFAIK doing the typedef like this means that the proto.Message won't be duplicated/copied. I mean this shouldn't have any overhead?